packages/e2e-tests/package.json

```diff
@@ -39,7 +39,10 @@
 	},
 	"peerDependencies": {
 		"jest": ">=27",
-		"puppeteer-core": ">=11"
+		"puppeteer": ">=11",
```

Comment:
Can we remove `puppeteer` here and land it?

Reply:
Yep, that sounds good.